request/process.go

```diff
@@ -23,14 +29,17 @@ func Process() {
 
 	switch request.Action {
 	case "configure":
-		break
+		configure(request)
```

Comment:
Doesn't this fall through and execute 'list' as well? Or does Go not require break statements?

Reply:
At last, we have a language that doesn't fall through by default. In go, if you want to fall through, you use keyword `fallthrough`.

Reply:
Finally! I like this :-).